app/models/manageiq/providers/vmware/infra_manager/metrics_capture.rb

```diff
@@ -237,23 +237,31 @@ def self.process_entity(data, parent = nil)
     ret
   end
 
+  attr_reader :objects, :ems
```

Comment:
Super-minor, but would this be clearer named `targets` to mirror the incoming target parameter?

Reply:
Yeah I can do that, I named it objects because that's what was [here](https://github.com/ManageIQ/manageiq-providers-vmware/pull/461/files#diff-5808adfc417996c7e2c77c9706c5a70aL273) but I can just update it in that method

Reply:
Done